README.md

```diff
@@ -1,30 +1,29 @@
-# Glutil
+# glutil
 
-A collection of utilities for managing AWS Glue partitions.
+A collection of utilities for managing partitions of tables in the AWS Glue Data Catalog that are built on datasets stored in S3.
 
 ## Hello Friends
 
-Glutil is currently in the process of being open sourced.
+`glutil` is currently in the process of being open sourced.
 There are still a couple small things we want to get done before calling that job done.
 As such, please pardon any sharp edges, and let us know about them by [creating an issue](https://github.com/Journera/glutil/issues/new).
 
 ## Background
 
-Amazon's Glue Catalog is generally pretty great, but sometimes things don't play well together, or a configuration mistake was made.
-For those cases, we have these utilities.
+AWS's Glue Data Catalog provides an index of the location and schema of your data across AWS data stores and is used to reference sources and targets for ETL jobs in AWS Glue. It is fully-integrated with AWS Athena, an ad-hoc query tool that uses the Hive metastore to build external tables on top of S3 data and PrestoDB to query the data with standard SQL. Journera heavily uses Kinesis Firehoses to write data from our platform to S3 in near real-time, Athena for ad-hoc analysis of data on S3, and Glue's serverless engine to execute PySpark ETL jobs on S3 data using the tables defined in the Data Catalog. Using the Data Catalog is generally pretty great, but sometimes all of these managed services don't play well together, or a configuration mistake was made (e.g., in a table DDL). For those cases, we have these utilities.
```

Comment:
I think our docs should use consistent formatting, which is one sentence per line, because when formatted markdown doesn't care about single line breaks and it can make navigating thoughts in a text editor and making comments on diffs easier.

----

I think the sentence starting with
> Journera heavily uses Kinesis Firehoses to
should be broken out into a separate paragraph because it's separate from what comes before (going from "what are these aws provided tools" to "this is how we use these tools")